src/Weather/LavaPiece.tscn

```diff
@@ -0,0 +1,23 @@
+[gd_scene load_steps=4 format=2]
+
+[ext_resource path="res://assets/art/blocks/demo_block.png" type="Texture" id=1]
+[ext_resource path="res://src/Weather/LavaPiece.gd" type="Script" id=2]
+
+[sub_resource type="RectangleShape2D" id=1]
+extents = Vector2( 16, 16 )
+
+[node name="LavaPiece" type="RigidBody2D"]
```

Comment:
Just curious - any particular reason for `RigidBody2D` rather than `KinematicBody2D`?

Reply:
Just seemed simpler, since RigidBody2Ds automatically apply gravity and other physics stuff. Since these didn't need to do things like move with input, I figured a RigidBody2D would get the job done.